### ISSUE TITLE
Does it work for you with  Google services v16?

### ISSUE THREAD

**Issue by Darkbiits** [f1xpl/openauto#181]
It's not working anymore for me :(

**Comment by Oper92** [comment]
Please check #170 and see if it solves the problem https://github.com/f1xpl/openauto/pull/170/commits/f05074e26758be7edac1208c7c3e7eb3e7292605

**Comment by Darkbiits** [comment]
No, at least it is not working for me when I connect the phone nothing happens.
can someone do a tutorial for the fix, I'm afraid to not do something right. Thanks

**Comment by parkerlreed** [comment]
That's a different issue. v17+ is broken as of right now.

https://github.com/f1xpl/aasdk/issues/32
